pkgs/data/themes/orchis-theme/default.nix

```diff
@@ -6,29 +6,33 @@
 , gtk-engine-murrine
 , sassc
 , border-radius ? null # Suggested: 2 < value < 16
-, tweaks ? [ ] # can be "solid" "compact" "black" "primary"
+, tweaks ? [ ] # can be "solid" "compact" "black" "primary" "macos" "submenu" "nord|dracula"
 , withWallpapers ? false
 }:
 
 let
-  validTweaks = [ "solid" "compact" "black" "primary" ];
-  unknownTweaks = lib.subtractLists validTweaks tweaks;
+  pname = "orchis-theme";
+
+  validTweaks = [ "solid" "compact" "black" "primary" "macos" "submenu" "nord|dracula" ];
```

Comment:
This list should contain both `nord` and `dracula`. The mutual exclusivity is ensured by `nordOrBlack`.

Reply:
You're right. I was distracted and didn't check the list when I copied it over from the tweaks comment.